tests/Controllers/CrudControllerTest.php

```diff
@@ -0,0 +1,15 @@
+<?php 
+namespace Afonso\LvCrud\Tests\Controllers;
+
+class CrudControllerTest extends BaseTestCase
```

Comment:
Doesn't this fail? `BaseTestCase` is outside the `tests/Controllers` folder, but looks like you're not `use`ing  it.